### ISSUE TITLE
Why not use MVVM?

### ISSUE THREAD

**Issue by Ibrahim-Islam** [toadlyBroodle/Listo#3]
Shouldn't using MVVM give the app a nice structure?

**Comment by toadlyBroodle** [comment]
This is my first c#, .NET app, so I'm on a pretty steep learning curve and haven't learned how or why to implement MVVM yet. What are the benefits of MVVM for a smaller, relatively simple application like this?

**Comment by Ibrahim-Islam** [comment]
I am not aware of the vision you have for the app so cant really speculate on whether MVVM should be appropriate but from your description sounded like it could be big and also MVVM gives structure and helpers to all size of apps.

**Comment by seboschtion** [comment]
I totally agree with Ibrahim-Islam, because small projects like this can grow very fast. It also simplifies collaborative work and will save you a lot of trouble and time when the project takes a step further (new features, bug fixes, etc...). For all the other advantages you can find more than enough ressources on the web, as you know.

But I understand your point, I asked me the same questions when I started developing. Just let me know where I can help.

**Comment by toadlyBroodle** [comment]
Ok thanks for your advice. When I have some time I will definitely invest it in learning MVVM! Since publishing this git project, I have actually changed the name of the app to Sublist (Listo was taken already in the Windows Store), and have also had to change the git project. So I will not be continuing to push to this project. I wasn't going to upload the new project, as there didn't seem to be much interest in this one, but if you are interested in working on it, I can start another. Right now I am having a problem with a most elusive bug that causes the app to hang on the splash screen and could most definitely use a hand rooting it out and implementing an MVVM structure :)

**Comment by Ibrahim-Islam** [comment]
Sure. I will try running it tonight and send you a pull request.

**Comment by toadlyBroodle** [comment]
That would be awesome. I've started a new repository with the latest version of the source code here: https://github.com/toadlyBroodle/Sublist
